docs/faq.rst

```diff
@@ -108,36 +79,35 @@ KA Lite can run on almost any major operating system: Windows, Linux, and Mac/OS
 What is data syncing?
 ---------------------
 
-KA Lite is capable to share your student progress data with a central data repository when you are online. This enables the system to have an online backup of your data, allows you to view your student progress online, and to share your data across multiple KA Lite installations.
+KA Lite is capable to share your student progress data with a `central data repository` when you are online. This enables the system to have an online backup of your data, allows you to view your student progress online, and to share your data across multiple KA Lite installations.
 
-Does KA Lite support peer to peer synchronization?
---------------------------------------------------
-
-Not yet. Peer to peer sync is a priority for KA Lite in the near future, but is not available yet.
+.. _central data https: https://hub.learningequality.org/
```

Comment:
Not gonna work since there's a name mismatch. I now prefer inline links in the paragraph, like this: 
```rst
`central data repository <https://hub.learningequality.org/>`_
```